common/screenshotter.go

```diff
@@ -175,27 +175,38 @@ func (s *screenshotter) screenshot(
 
 	// Add clip region
 	//nolint:dogsled
-	_, visualViewport, _, _, _, _, err := cdppage.GetLayoutMetrics().Do(cdp.WithExecutor(s.ctx, sess))
+	_, _, _, _, visualViewport, _, err := cdppage.GetLayoutMetrics().Do(cdp.WithExecutor(s.ctx, sess))
```

Comment:
> In theory, we can also try to set values from deprecated visualViewport if that's not nil and current visualViewport isn't defined 🤔 WDYT?

I'm personally happy to drop the deprecated `visualViewport` and only work with `cssVisualViewport`. I'm assuming the values are the same, it's just a field name change?

Reply:
Ah, i see 🤔 

At the moment we don't have a way to map a k6 version to a specific chrome version, and therefore we haven't documented it anywhere what guarantees we have in place in terms of k6 to chrome version, and when we stop/start ending support for a specific chrome version. So in some way, we're either suggesting k6 browser works with "all" chrome versions from a specific point, which could be when we created k6 browser or when it became an experimental module.

We have had requests for k6 browser to work with "very old" versions of chrome (2+ years old).

What about if we work with both `visualViewport` and `cssVisualViewport`? So work with `cssVisualViewport`, and fallback to `visualViewport`? This only so that we don't introduce a breaking change for anyone that is relying on working against chrome 91 (or less).

WDYT?

Reply:
>What about if we work with both visualViewport and cssVisualViewport? So work with cssVisualViewport, and fallback to visualViewport? This only so that we don't introduce a breaking change for anyone that is relying on working against chrome 91 (or less).

Yep, that was my original suggestion for this tread started message, so we can do it :+1: